static/app/views/releases/list/index.tsx

```diff
@@ -345,75 +394,129 @@ export default function ReleasesList() {
-                  onSearch={handleSearch}
-                  initialQuery={activeQuery}
-                  filterKeys={RELEASE_FILTER_KEYS}
-                  getTagValues={getTagValues}
-                  placeholder={t('Search by version, build, package, or stage')}
-                  searchSource="releases"
-                />
-                <ReleasesStatusOptions selected={activeStatus} onSelect={handleStatus} />
-                <ReleasesSortOptions
-                  selected={activeSort}
-                  selectedDisplay={activeDisplay}
-                  onSelect={handleSortBy}
-                  environments={selection.environments}
-                />
-                <ReleasesDisplayOptions
-                  selected={activeDisplay}
-                  onSelect={handleDisplay}
-                />
-              </SortAndFilterWrapper>
+            {shouldShowMobileBuildsTab && (
+              <Layout.HeaderTabs value={selectedTab}>
+                <TabList aria-label={t('Releases tab selector')}>
+                  <TabList.Item
+                    key="releases"
+                    to={{
+                      pathname: location.pathname,
+                      query: {...location.query, tab: undefined},
+                    }}
+                    textValue={t('Releases')}
+                  >
+                    {t('Releases')}
+                  </TabList.Item>
+                  <TabList.Item
+                    key="mobile-builds"
+                    to={{
+                      pathname: location.pathname,
+                      query: {...location.query, tab: 'mobile-builds'},
+                    }}
+                    textValue={t('Mobile Builds')}
+                  >
+                    <Flex align="center" gap="sm">
+                      {t('Mobile Builds')}
+                      <FeatureBadge type="beta" />
+                    </Flex>
+                  </TabList.Item>
+                </TabList>
+              </Layout.HeaderTabs>
             )}
-
-            {!(isReleasesPending || isReleasesRefetching) &&
-              activeStatus === ReleasesStatusOption.ARCHIVED &&
-              !!releases?.length && <ReleaseArchivedNotice multi />}
-
-            {releasesErrorMessage ? (
-              <LoadingError message={releasesErrorMessage} />
-            ) : (
-              <DemoTourElement
-                id={DemoTourStep.RELEASES_LIST}
-                title={t('Latest releases')}
-                description={t(
-                  'View the latest releases for your project. Select a release to review new and regressed issues, and business critical metrics like crash rate, and user adoption. '
-                )}
-                position="top-start"
-              >
-                <ReleaseListInner
-                  activeDisplay={activeDisplay}
-                  loading={isReleasesPending}
+          </Stack>
+        </Layout.Header>
+        <Layout.Body data-tab={selectedTab}>
```

Comment:
does this power anything?

```suggestion
        <Layout.Body>
```

Reply:
Don't believe so, will remove